webpack/assets/javascripts/react_app/redux/reducers/status/index.js

```diff
@@ -0,0 +1,19 @@
+import Immutable from 'seamless-immutable';
+import { STATUS_REQUEST, STATUS_SUCCESS, STATUS_FAILURE } from '../../consts';
+
+const initialState = Immutable({});
+
+export default (state = initialState, action) => {
+  const { payload } = action;
+
+  switch (action.type) {
+    case STATUS_REQUEST:
+      return state;
+    case STATUS_SUCCESS:
+      return state.set(payload.id, { ...payload });
```

Comment:
I'm a bit worried that ID's can override each other, while I've seen you've prefixed it, the fact is that these actions are used for multiple objects, can confuse a future developer?